File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        s = ''
        sizes = [len(word1), len(word2)]
        for i in range(min(sizes)):
            s = s + word1[i]
            s = s + word2[i]
        if(sizes[0] < sizes[1]):
            s = s + word2[sizes[0]:]
        else:
            s = s + word1[sizes[1]:]
        return s 

File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string result = "";
        int minIndex = min(word1.length(), word2.length());
        for (int i = 0; i < minIndex; i++) {
            result += word1[i];
            result += word2[i];
        }
        if (word1.length() < word2.length()) {
            for (int i = minIndex; i < word2.length(); i++)
                result += word2[i];
        } else if (word2.length() < word1.length()) {
            for (int i = minIndex; i < word1.length(); i++)
                result += word1[i];
        }
        return result;
    }
};